src/main.c

```diff
@@ -26,7 +26,17 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-//-----------------------------------------------------------------------------
+// ----------------------------------------------------------------------------
+// We need 240 CV values
+// #define LOCONET_CV_NUMBERS 240
+
+#define DOMOTICA_RX_INPUT_ADDRESS_SIZE 6
```

Comment:
Multiple definitions of this value. You've set a default in `domotica.h`, and you define it twice in `main.c`, with the same value as the default. Please remove the defines from `main.c` to allow overriding using `Makefile.make`.

Same goes for `DOMOTICA_FAST_CLOCK_SIZE` and `DOMOTICA_LNCV_START_OUTPUT_BRIGHTNESS`. These are also defined multiple times.